docs/publishing.md

```diff
@@ -117,4 +125,6 @@ from Atlassian on the flow. These steps are manual because you might encounter m
    development. This is because the `npm run vscode:package` step does a `npm
    prune --production`. This is required due to the way Lerna does symlinking.
    See [vscode-vsce#52](https://github.com/Microsoft/vscode-vsce/issues/52) for
-   more information.
\ No newline at end of file
+   more information.
+
+[publish_vscode_ext]: https://code.visualstudio.com/docs/extensions/publish-extension
```

Comment:
Usually we don't use this style of including links but since this one is referenced in two places, this seems like a good exception.